sites/public/pages/about.tsx

```diff
@@ -0,0 +1,82 @@
+import React from "react"
+import { MarkdownSection, t, PageHeader, GridSection, GridCell } from "@bloom-housing/ui-components"
+import Layout from "../layouts/application"
+import PageContent from "../page_content/about.mdx"
+
+export default function About() {
+  return (
+    <Layout>
+      <PageHeader title={t("pageTitle.about")} subtitle={t("pageTitle.aboutSubtitle")} inverse />
```

Comment:
Looks like these need associated text+translations:

![image](https://user-images.githubusercontent.com/8754454/134077689-2e68c9b0-cde5-4bfb-be5e-794bf6488bd3.png)

Reply:
Whoops, that was one of those things I meant to do before opening the PR!